package.json

```diff
@@ -162,7 +163,7 @@
     "test-unit": "node test/pretest.js && mocha",
     "test-admin": "mocha --opts test/mocha-admin.opts",
     "test-e2e": "node test/e2e/server.js --env default",
-    "test-e2e-bg": "export KNE_SELENIUM_START_PROCESS=false && node test/e2e/server.js --env default",
+    "test-e2e-bg": "cross-env KNE_SELENIUM_START_PROCESS=false && node test/e2e/server.js --env default",
```

Comment:
fyi...we don't really need to use an env var export here as there's a `--selenium-in-background` switch that can be specified.